website/content/en/docs/upgrading/compatibility.md

```diff
@@ -86,6 +86,16 @@ By adopting this practice we allow our users who are early adopters to test out
 
 ### Snapshot Releases
 
-We release a snapshot release for every commit that gets merged into the main repository. This enables our users to immediately try a new feature or fix right after it's merged rather than waiting days or weeks for release.
-Snapshot releases are suitable for testing, and troubleshooting but users should exercise great care if they decide to use them in production environments.
-Snapshot releases are tagged with the git commit hash prefixed by the Karpenter major version. For example `v0-fc17bfc89ebb30a3b102a86012b3e3992ec08adf`. For more detailed examples on how to use snapshot releases look under "Usage" in [Karpenter Helm Chart](https://gallery.ecr.aws/karpenter/karpenter).
+We release a snapshot release for every commit that gets merged into the [`aws/karpenter-provider-aws`](https://www.github.com/aws/karpenter-provider-aws). This enables our users to immediately try a new feature or fix right after it's merged rather than waiting days or weeks for release.
+Snapshot releases are not made available in the same public ECR repository as other release types, they are instead published to a separate private ECR repository.
+Helm charts are published to `oci://021119463062.dkr.ecr.us-east-1.amazonaws.com/karpenter/snapshot/karpenter` and are tagged with the git commit hash prefixed by the Karpenter major version (e.g. `v0-fc17bfc89ebb30a3b102a86012b3e3992ec08adf`).
+Anyone with an AWS account can pull from this repository, but must first authenticate:
+
+```bash
+aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 021119463062.dkr.ecr.us-east-1.amazonaws.com
+```
+
+{{% alert title="Note" color="warning" %}}
+Snapshot releases are suitable for testing, and troubleshooting but users should exercise great care if they decide to use them in production environments. Additionally, these releases are ephemeral and will be removed 90 days after they were published.
```

Comment:
I wouldn't even say "exercise great care". I would just outright say that we do not support these in production environments.